lib/bolt/inventory/inventory2.rb

```diff
@@ -208,12 +217,15 @@ def set_facts(tname, hash)
       end
       private :set_facts
 
-      def add_node(current_group, target, desired_group, track = { 'all' => nil })
+      def add_target(current_group, target, desired_group, track = { 'all' => nil })
         if current_group.name == desired_group
           # Group to add to is found
           t_name = target.name
-          # Add target to nodes hash
-          current_group.nodes[t_name] = { 'name' => t_name }.merge(target.options)
+          # Add target to targets hash
+          target_hash = { 'name' => t_name }.merge(target.options)
```

Comment:
This change fixes a bug previously introduced where adding a new target to a group using just a string name caused the URI to not be set.